### PR TITLE
METAGEN-85

### PR DESCRIPTION
**Pull request by gunnarmorling**

### CODE REVIEW

File: src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java [hunk 1]
```diff
@@ -252,7 +252,7 @@ private void handleRootElementAnnotationMirrors(final Element element) {
 			metaEntity = new AnnotationMetaEntity( (TypeElement) element, context, requiresLazyMemberInitialization );
 
 			if ( alreadyExistingMetaEntity != null ) {
-				metaEntity.mergeInMembers( alreadyExistingMetaEntity.getMembers() );
+				metaEntity.mergeInMembers( alreadyExistingMetaEntity );
```
Comment (hferentschik) [inline src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java:255]: Exactly that was my thinking as well. Pass the whole entity. :-)

File: src/main/java/org/hibernate/jpamodelgen/xml/XmlMetaEntity.java [hunk 2]
```diff
@@ -360,7 +360,7 @@ private String getType(String propertyName, String explicitTargetEntity, Element
 		context.logMessage(
 				Diagnostic.Kind.WARNING,
 				"Unable to determine type for property " + propertyName + " of class " + getQualifiedName()
-						+ " using assess type " + accessTypeInfo.getDefaultAccessType()
+						+ " using access type " + accessTypeInfo.getDefaultAccessType()
```
Comment (hferentschik) [inline src/main/java/org/hibernate/jpamodelgen/xml/XmlMetaEntity.java:363]: lol

File: src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java (hunk 1 above)
Comment (gunnarmorling) [inline src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java:255]: Yeah, in the end the change was quite simple. Just needed to find it :-)

**Comment by hferentschik** [comment]
Nice. Passing all tests on my Ubuntu test system now :-)

**Comment by hferentschik** [comment]
I guess I can cut a release tomorrow :-)

File: src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java (hunk 1 above)
Comment (hferentschik) [inline src/main/java/org/hibernate/jpamodelgen/JPAMetaModelEntityProcessor.java:255]: Good stuff, hope you liked the little METAGEN diversion.

**Comment by gunnarmorling** [comment]
> I guess I can cut a release tomorrow :-)

+1 Sounds good.

**Comment by gsmet** [comment]
I confirm everything is OK here too:
Tests run: 61, Failures: 0, Errors: 0, Skipped: 0

Thanks!

**Comment by hferentschik** [comment]
@gsmet Thanks for double checking

**Pull request merged**